### DISCUSSION TITLE
[Config Support]: Alternate required zones

### DISCUSSION THREAD

**Discussion by turboc1208** [blakeblackshear/frigate#10339] [Config Support]
### Describe the problem you are having

I have a camera pointing at the parking area in my driveway.  There are two valid ways to get to the parking area.  In the documentation there is an example requiring that a car pass through zone 1 and it stops recording when the car gets into zone2.  Is there a way to setup two required zones (zone 1 from the left and zone 2 from the right both leading into zone 3 in the middle making zone 1 OR zone 2 required?

### Version

0.13.2-6476F8A

### Frigate config file

```yaml
NA
```

### Relevant log output

```shell
NA
```

### Frigate stats

```json
NA
```

### Operating system

Debian

### Install method

Docker Compose

### Coral version

USB

**Comment by NickM-27** [comment]
When you make a list of required zones, the list is OR
